packages/api-schema/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
+node_modules
```

Comment:
Instead of adding a `.gitignore`, could we move them to the top-level `.gitignore`?

Reply:
Okay. No strong opinions about this. Let's keep it as is.